l10n_it_fattura_accompagnatoria/models/account_invoice.py

```diff
@@ -1,5 +1,5 @@
-# -*- coding: utf-8 -*-
 # Copyright 2017 Lorenzo Battistini - Agile Business Group
+# Copyright 2020 Simone Vanin - Agile Business Group
 # License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
 
 
```

Comment:
È ancora vero quanto scritto qui sotto o valeva solo in v10?
Mi potresti chiarire a cosa serve il nuovo campo `note`?

Reply:
Il campo `note` di account.invoice è uguale al campo `comment`. Non dovrebbe più servire perciò lo elimino.

Reply:
Chiarisco come funziona il modulo: il template `delivery_data` viene chiamato passandogli la fattura e stampando quindi i dati della fattura. Il campo `note` veniva aggiunto perchè serve a `delivery_data`; infatti il campo `note` è `related="comment"`

Reply:
Grazie @eLBati del chiarimento, allora @SimoneVagile può essere che il campo sia da ripristinare dopo la modifica suggerita in https://github.com/OCA/l10n-italy/pull/1908#discussion_r503736622, puoi verificare?

Reply:
@eLBati @SimoRubi se è verificato ciò che affermo in [https://github.com/OCA/l10n-italy/pull/1908#discussion_r503800290](https://github.com/OCA/l10n-italy/pull/1908#discussion_r503800290) il campo note su `account.invoice` risulta superfluo perché è nel model dei ddt.

Reply:
Si può risolvere questo commento? Il codice ora è a posto.